app/helpers/adoption_applications_helper.rb

```diff
@@ -0,0 +1,58 @@
+module AdoptionApplicationsHelper
+  def us_states
+    [
+      ["Alabama", "AL"],
+      ["Alaska", "AK"],
+      ["Arizona", "AZ"],
+      ["Arkansas", "AR"],
+      ["California", "CA"],
+      ["Colorado", "CO"],
+      ["Connecticut", "CT"],
+      ["Delaware", "DE"],
+      ["District of Columbia", "DC"],
+      ["Florida", "FL"],
+      ["Georgia", "GA"],
+      ["Hawaii", "HI"],
+      ["Idaho", "ID"],
+      ["Illinois", "IL"],
+      ["Indiana", "IN"],
+      ["Iowa", "IA"],
+      ["Kansas", "KS"],
+      ["Kentucky", "KY"],
+      ["Louisiana", "LA"],
+      ["Maine", "ME"],
+      ["Maryland", "MD"],
+      ["Massachusetts", "MA"],
+      ["Michigan", "MI"],
+      ["Minnesota", "MN"],
+      ["Mississippi", "MS"],
+      ["Missouri", "MO"],
+      ["Montana", "MT"],
+      ["Nebraska", "NE"],
+      ["Nevada", "NV"],
+      ["New Hampshire", "NH"],
+      ["New Jersey", "NJ"],
+      ["New Mexico", "NM"],
+      ["New York", "NY"],
+      ["North Carolina", "NC"],
+      ["North Dakota", "ND"],
+      ["Ohio", "OH"],
+      ["Oklahoma", "OK"],
+      ["Oregon", "OR"],
+      ["Pennsylvania", "PA"],
```

Comment:
I think these are fine as is but if you'd like to follow Hound's advice this would be:
```rb
%w(Pennsylvania PA)
```